package.json

```diff
@@ -48,6 +48,12 @@
     "@commitlint/config-conventional": "^17.3.0",
     "@dwarvesf/react-eslint-config": "^1.0.1",
     "@prettier/plugin-xml": "^3.2.0",
+    "@babel/core": "^7.22.5",
+    "@babel/helper-plugin-utils": "^7.20.2",
+    "@babel/preset-react": "^7.22.5",
```

Comment:
Do we still need those 3 packages? I think they have been removed in https://github.com/dwarvesf/nextjs-boilerplate/commit/7898e9d72b93d1a3e88ecc4814bff8fbddfd834c

Reply:
seems like it's a conflict while I merged code. Just removed